packages/big-design/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -369,7 +369,7 @@ export const MultiSelect = typedMemo(
             }))}
             iconRight={renderToggle}
             readOnly={!filterable}
-            required={required}
+            required={required && selectedOptions.length === 0 ? true : false}
```

Comment:
🍹 
```suggestion
            required={required && selectedOptions.length === 0}
```

Reply:
Doh, good idea! Initially I had a different logic so I kept the conditional 🤦